src/SIUnits.jl

```diff
@@ -7,15 +7,28 @@ module SIUnits
     import Base: ==, +, -, *, /, .+, .-, .*, ./, //, ^
     import Base: promote_rule, promote_type, convert, show, mod
 
-    immutable SIQuantity{T<:Number,m,kg,s,A,K,mol,cd,rad,sr} <: Number
+    typealias UnitTuple NTuple{9,Int64}
```

Comment:
Actually, I was quite sure it *should* work (in fact, IIRC the move from ints-and-types as (values-of-any-bitstypes)-and-types as valid type parameters was made in base Julia with e.g. `s^(1/2)` as the main use case), but when I try now, it doesn't:

```
julia> using SIUnits; Second^(1/2)
1.0
```

Don't know if this is a regression or if it never worked, though.